src/lib/PetitioRequest.ts

```diff
@@ -3,16 +3,15 @@
  */
 import type { DispatchOptions as ADO, Options as AO } from "undici/types/agent";
 import { Agent, Client } from "undici";
-import type AbortController from "node-abort-controller";
 import type { Options as CO } from "undici/types/client";
 import type { DispatchOptions as DO } from "undici/types/dispatcher";
-import type { IncomingHttpHeaders } from "http";
-import type { ParsedUrlQueryInput } from "querystring";
+import type { IncomingHttpHeaders } from "node:http";
+import type { Readable } from "node:stream";
+import { URL } from "node:url";
+import { join } from "node:path";
+// eslint-disable-next-line sort-imports
+import { type ParsedUrlQueryInput, stringify } from "node:querystring";
```

Comment:
Is there any particular reason to disable the check instead of simply sorting the imports here?